x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/field_deprecation_warning/field_deprecation_warning.scss

```diff
@@ -0,0 +1,13 @@
+.ilmFieldDeprecationWarning {
```

Comment:
I'll take a look locally, but these negative margins have me wondering how well this will hold up over time/cross-browser/responsively. Perhaps we could avoid floating the icon to the left of the input by using a prepend or append? https://elastic.github.io/eui/#/forms/form-controls#prepend-and-append

Reply:
After checking this out a bit more, I do think we should take a different approach here. At smaller sizes (granted its not the most common use case) the warning is not visible. Further, since this triggers for any value, then we should just show it by default / all the time. The prepend or append approach achieves this result.

Reply:
Hey @ryankeairns ! Thanks for the review and checking out the code locally, happy to make adjustments per your recommendations.

> Further, since this triggers for any value, then we should just show it by default / all the time

I was considering this in the initial implementation, but I was concerned that always having the yellow alert logo visible would draw the users attention where it may not be needed. I'd be curious to get your thoughts on this:

<img width="954" alt="Screenshot 2021-04-14 at 15 44 36" src="https://user-images.githubusercontent.com/8155004/114720610-73b67980-9d38-11eb-87a2-f82a46ff69d4.png">

We could also show the prepend when the user enters a value since I don't think the pop in effect is particularly disorientating in this case:

![popin-max-index-size](https://user-images.githubusercontent.com/8155004/114721709-7bc2e900-9d39-11eb-8ddd-5103aba7258b.gif)


CC @mdefazio

Reply:
Thanks for trying this out. I'll defer to DeFazio on the final adjustments.